linera-base/src/identifiers.rs

```diff
@@ -50,6 +50,15 @@ impl AccountOwner {
     pub fn is_chain(&self) -> bool {
         self == &AccountOwner::CHAIN
     }
+
+    /// The size of the `AccountOwner`.
+    pub fn size(&self) -> u32 {
+        match self {
+            AccountOwner::Reserved(_) => 1,
+            AccountOwner::Address32(_) => 32,
+            AccountOwner::Address20(_) => 20,
```

Comment:
technically + 1 for the tag

Reply:
maybe we should just count `32` to simplify, though

Reply:
Using 32 was indeed my guess.
I thought that putting 32 was an error since we may have addresses larger than 32 with this post-quantum stuff.

About the tag, yes, it is there. But we want something easy to understand by programmers.